flake.nix

```diff
@@ -40,6 +38,8 @@
     '';
   in {
     lib = {
+      inherit moduleMod mkHackageMod;
```

Comment:
Given these are unversioned, they are unstable. We need a better naming scheme possibly though. A lib.unstable?